frontend/packages/dev-console/src/components/topology/componentUtils.ts

```diff
@@ -170,10 +176,35 @@ const groupWorkoadDropTargetSpec: DropTargetSpec<
   }),
 };
 
+const graphEventSourceDropTargetSpec: DropTargetSpec<
+  GraphElement,
```

Comment:
Since `MOVE_EV_SRC_CONNECTOR_DROP_TYPE` is the only accepted type, and it always supplies an item of type `Edge` we can do this:
```suggestion
  Edge,
```